Common/Data/SubscriptionManager.cs

```diff
@@ -209,7 +209,8 @@ public void AddConsolidator(Symbol symbol, IDataConsolidator consolidator)
                 {SecurityType.Cfd, new List<TickType>() { TickType.Quote } },
                 {SecurityType.Future, new List<TickType>() { TickType.Quote, TickType.Trade, TickType.OpenInterest } },
                 {SecurityType.Commodity, new List<TickType>() { TickType.Trade } },
-                {SecurityType.Crypto, new List<TickType>() { TickType.Quote } }
+                {SecurityType.Crypto, new List<TickType>() { TickType.Quote } },
+                {SecurityType.Crypto, new List<TickType>() { TickType.Trade } }
```

Comment:
Should be: 
`{SecurityType.Crypto, new List<TickType>() { TickType.Quote, TickType.Trade } }`

Reply:
Yes, so this is the first step of a fairly significant global change and goes beyond anything I had planned.  How are you imagining the rest of this playing out?

Reply:
I'm assuming the global change you mention is inside the GDAX brokerage implementation, not in LEAN engine ?
Multiple tick types for a security type (and their subscriptions) are already supported inside LEAN (currently enabled only for Futures and Options, only brokerage implementation that has them is IB). 
Since crypto brokerages like GDAX make both quotes and trades available, I think it would be nice to have them both included in the initial version, the full OrderBook should be considered a new feature to be added later on (requiring new data types in LEAN).

Reply:
That's great. You were talking about introducing multiple tick types when we floated the bitfinex brokerage. I didn't realize it had been done since then. You guys have been busy.